components/compass-runtime-agent/internal/compass/director/converter_test.go

```diff
@@ -643,26 +561,6 @@ func fixCompassDocument(suffix string, data *graphql.CLOB) *graphql.Document {
 	}
 }
 
-func fixCompassODataSpec() *graphql.APISpec {
-	data := graphql.CLOB(`OData spec`)
-
-	return &graphql.APISpec{
-		Data:   &data,
-		Type:   graphql.APISpecTypeOdata,
-		Format: graphql.SpecFormatXML,
-	}
-}
-
-func fixCompassOpenAPISpec() *graphql.APISpec {
-	data := graphql.CLOB(`Open API spec`)
-
-	return &graphql.APISpec{
-		Data:   &data,
-		Type:   graphql.APISpecTypeOpenAPI,
-		Format: graphql.SpecFormatJSON,
-	}
-}
-
 func fixCompassAsyncAPISpec() *graphql.EventSpec {
```

Comment:
I believe functions fixCompassAsyncAPISpec and  fixCompassDocContent should also be removed. We don't fetch this data from Compass anymore.